util/nrt_diff/iidmDiff.py

```diff
@@ -147,9 +150,10 @@ def OutputIIDMCloseEnough (path_left, path_right):
                                     continue
                             nb_differences+=1
                             msg += "[ERROR] attribute " + attr1 + " of object " + firstId + " (type " + firstObj.type +") value: " + firstObj.values[attr1] + " has another value on right side (value: " + secondObj.values[attr1] + ")\n"
-
             for attr1 in secondObj.values:
                 if attr1 not in firstObj.values:
                     nb_differences+=1
                     msg += "[ERROR] attribute " + attr1 + " of object " + firstId + " (type " + firstObj.type +") value: " + secondObj.values[attr1] + " is not in the equivalent object on left side\n"
+    for error in sorted(differences, key=operator.itemgetter(0), reverse=True)[:settings.max_nb_iidm_outputs]:
```

Comment:
Does this work in the length of the table is below 10? I think there is a unit test which is in this case

Reply:
To be honest, I have just trusted that Python handled it for me (optimistic hypothesis I agree ;)) but anyway, the unit test doesn't work because I should update a reference somewhere so I will be able to check this.

Reply:
Unit test updated, apparently working when table length is lower than 10.